CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.5.4 / 2019-01-01
```

Comment:
I added current date, if it is accepted but released in another day, it has to be updated.